sdk/eventhub/event-processor-host/changelog.md

```diff
@@ -1,3 +1,24 @@
+## 2019-07-16 2.0.0
+- Use the latest version of the dependency on [@azure/event-hubs](https://www.npmjs.com/package/@azure/event-hubs/v/2.1.1) that has fixes for the following bugs
+      - Added event handlers for `error` and `protocolError` events on the connection object to avoid the case of unhandled exceptions. This is related to the [bug 4136](https://github.com/Azure/azure-sdk-for-js/issues/4136)
+      - A network connection lost error is now treated as retryable error. A new error with name `ConnectionLostError` 
+      is introduced for this scenario which you can see if you enable the [logs](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs#enable-logs).
```

Comment:
Can EPH users use the same kind of logging mechanism? Should we point them to the section in EPH instead of EH?

Reply:
I also thought about it but above we mentioned that event hubs has the following bug fixes that's why I added the link for event hubs logging mechanism.

Event hubs logging mechanism is different than EPH:
Event hubs : `export DEBUG=azure*`
EPH: `export DEBUG=azure:eph*`

Reply:
I'll update the link to point them to the EPH debug section.